app/javascript/components/applications/application_form/PaperApplicationForm.js

```diff
@@ -79,8 +81,11 @@ class PaperApplicationForm extends React.Component {
     if (values.demographics) {
       const { demographics } = values
       errors.demographics = {}
-      errors.demographics.gender_other = this.checkNotListed(demographics, 'gender', 'Gender is required')
-      errors.demographics.sexual_orientation_other = this.checkNotListed(demographics, 'sexual_orientation', 'Sexual Orientation is required')
+
+      errors.demographics.gender_other = this.checkNotListed(demographics, 'gender') ? 'Gender is required' : undefined
+      this.state.genderSpecifyRequired = this.checkNotListed(demographics, 'gender')
```

Comment:
When setting state here we should still use `this.setState` instead of direct assignment. This is more stable with our components lifecycle methods. One easy way could be to do something like
``` js
const isGenderSpecifyRequired = this.checkNotListed(demographics, 'gender');
const isOrientationOtherRequired = this.checkNotListed(demographics, 'sexual_orientation');
errors.demographics.gender_other = isGenderSpecifyRequired ? 'Gender is required' : undefined
errors.demographics.sexual_orientation_other = isOrientationOtherRequired ? 'Sexual Orientation is required' : undefined
this.setState({ genderSpecifyRequired: isGenderSpecifyRequired, orientationOtherRequired: isOrientationOtherRequired })
```

Reply:
update made!